libr/debug/debug.c

```diff
@@ -95,8 +97,13 @@ static int r_debug_bp_hit(RDebug *dbg, RRegItem *pc_ri, ut64 pc, RBreakpointItem
 	dbg->reason.bp_addr = b->addr;
 
 	/* inform the user of what happened */
-	eprintf ("hit %spoint at: %"PFMT64x "\n",
-			b->trace ? "trace" : "break", pc);
+	if (dbg->corebind.config) {
+		showinfo = r_config_get_i (dbg->corebind.config, "cmd.hitinfo");
```

Comment:
can you give me a pointer on where in the code i should add this callback? i guess it should be where the RDebug structure is initialized. I also see the possible danger that if the RDebug structure is freed, then at that point the callback should also be removed. if it gets too complex to implement, i might drop this feature in this commit and think about it for a future commit.